src/utils/viewer/viewer.cpp

```diff
@@ -783,8 +784,12 @@ void Viewer::keyPressEvent(KeyEvent& event) {
           !simulator_->isNavMeshVisualizationActive());
       break;
     case KeyEvent::Key::I:
-      Mn::DebugTools::screenshot(Mn::GL::defaultFramebuffer,
-                                 "test_image_save.png");
+      if (!Cr::Utility::Directory::exists("screenshots/")) {
+        Cr::Utility::Directory::mkpath("screenshots/");
+      }
+      Mn::DebugTools::screenshot(
+          Mn::GL::defaultFramebuffer,
+          "screenshots/" + std::to_string(savedFrames++) + ".png");
```

Comment:
Nit: it'd be nice if this feature didn't overwrite existing screenshots from previous runs. Can you think of any easy way to improvise unique filenames? My Linux screenshot tool uses timestamps.

Reply:
@eundersander: An int64 number as the suffix? The largest number means the latest screenshot? :)

Reply:
int64??? We expect people to take 1 billion+ images in a single folder. Most setups wouldn't have that many inodes across the whole filesystem.

Reply:
I'm now caching the date-time when viewer is started. This is used to create a directory for screenshots to keep them separated on each run.